tests/test_config.py

```diff
@@ -26,7 +32,7 @@ def get_GMXRC():
     bindir = os.path.dirname(path)
     GMXRC = os.path.join(bindir, 'GMXRC')
     if not os.path.exists(GMXRC):
-        raise IOError(errno.EEXIST, "Could not find Gromacs setup file", GMXRC)
+        raise IOError(errno.ENOENT, "Could not find Gromacs setup file", GMXRC)
```

Comment:
This now raises the error that the file does not exist instead of that it does exist (it would be returned if it did exist).

Reply:
oops. thanks for catching